pkg/steps/release/create_release.go

```diff
@@ -204,7 +204,19 @@ done
 if [[ "$exit_code" != "0" ]]; then
 	exit $exit_code
 fi
-oc adm release extract --from=%q --to=${ARTIFACT_DIR}/release-payload-%s
+for ((i=1; i<=5; i++)); do
+	if oc adm release extract --from=%q --to=${ARTIFACT_DIR}/release-payload-%s; then
```

Comment:
Do you know if it's a problem on retry if `${ARTIFACT_DIR}/release-payload-%s` still has contents in it?

Reply:
I tried running it locally with partial contents and there was no error